client/placeholder/dist/index.html

```diff
@@ -0,0 +1,51 @@
+<html>
```

Comment:
suggestion: No doubt this renders, but we should probably make it a more complete HTML document for accessibility purposes. That feels like a reasonable follow-up given the role this has.

Thinking page title and img alt text primarily. Suggest pointing lighthouse-ci at it as a quick a11y check.